<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set</class>
 <widget class="QWidget" name="Set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="ImagePath_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="IP_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>服务器IP地址：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="esc_butt">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>智慧停车系统设置界面</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>服务器端口号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_butt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>xxx.xxx.xxx.xxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImagePath_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>车牌图片保存地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="VideoPath_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>视频录制保存地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="VideoPath_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoPath_butt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImagePath_butt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
